addons/danger/settings.inc.sqf

```diff
@@ -108,6 +108,70 @@ _curCat = LSTRING(Settings_GeneralCat);
     1
 ] call CBA_fnc_addSetting;
 
+addMissionEventHandler ["GroupCreated", {
+    params ["_group"];
+    [
+        {
+            if (count (units _this) > 0) then {
+                if (GVAR(autoAddDynamicReinforcement)) then {_this setVariable [QGVAR(enableGroupReinforce), true, true];};
```

Comment:
i think we maybe should split this up into a more logical system. why should we, for example, wait when all 3 settings are disabled?

Reply:
It's a 3-second wait before executing ([`waitAndExecute`](https://github.com/CBATeam/CBA_A3/blob/master/addons/common/fnc_waitAndExecute.sqf)) instead of waiting until all 3 are enabled ([`waitUntilAndExecute`](https://github.com/CBATeam/CBA_A3/blob/master/addons/common/fnc_waitUntilAndExecute.sqf)), but that's still a good point.